notebooks/00_quick_start/sar_movielens_with_azureml.ipynb

```diff
@@ -271,14 +225,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Found existing compute target\n"
+      "Creating a new compute target...\n",
```

Comment:
i think both terms are present because they are representing slightly different concepts: the compute target is the specific place where the notebook will be executed, and in this case we are using AzureML Compute (the service) as that target. Alternatively the compute target could be the local machine, a VM, or HDInsight. docs here: https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-set-up-training-targets#amlcompute

we should clean up references to aml though

Reply:
Hey, I asked PMs about terminology and the recommendation is to use AmlCompute until they decide on term. I'll create a PR for this. Tagged you both in Teams